speech/cloud-client/beta_snippets_test.py

```diff
@@ -42,3 +44,11 @@ def test_transcribe_file_with_auto_punctuation(capsys):
     out, _ = capsys.readouterr()
 
     assert 'Okay. Sure.' in out
+
+
+def test_transcribe_diarization(capsys):
+    transcribe_file_with_diarization(
+        os.path.join(RESOURCES, 'Google_Gnome.wav'))
+    out, err = capsys.readouterr()
+
+    assert 'OK Google stream stranger things from Netflix to my TV' in out
```

Comment:
are there two different speakers in this audio file?  if so assert something about the speaker_tags being returned correctly.